src/pages/Calculator/Steps/SeedingMethod/index.js

```diff
@@ -246,55 +256,62 @@ const SeedingMethod = () => {
                 {council === 'MCCC'
                   && (
                   <>
-                    {renderMethod('Precision', methods[seed.label]?.Precision)}
-                    {renderMethod('Drilled', methods[seed.label]?.Drilled)}
+                    {renderMethod('Precision', methods[seed.label]?.Precision, '', 'Precision')}
```

Comment:
why did you add another param with the same value? and why another param with empty string?

Reply:
In Seeding Method page, The first param is the Bolded seeding method at left(`Precision`, `Drilled`, etc), the second param is the value at right, the third param is the text under the seeding method(`with Light Incorporation`, etc), the fourth param is the selected seeding method(the method in dropdown).

This function do contains many duplicate params but it needs to handle different situations in councils, I can reduce the length of the param by reusing the method name in second param.